draft-demarco-oauth-nonce-endpoint.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "OAuth 2.0 Nonce Endpoint"
```

Comment:
I like this change.

Reply:
this PR must be mapped on separate issues and PRs, where there's the agreement about making this specification more general and not too much tied on OAuth 2.0

at the same time I'm wondering if this might have impact in the inclusion in some or other IETF WGs. We will discuss on this.